### ISSUE TITLE
Detailed split of the dataset

### ISSUE THREAD

**Issue by Fyc-Ethan** [HKUSTDial/VisJudgeBench#2]
Dear Author,
@xypkent 

In your paper, you mentioned using VISJUDGE-BENCH’s human-annotated data with a 70%/10%/20% train/validation/test split (2,163/279/648 samples).

But after I downloaded your dataset, I found that I couldn't tell the difference between the test set and the training set. Could you please share some detailed information about your dataset splitting?
At the same time, I noticed that the number of images in your dataset appears to be larger than the number mentioned in the paper. Could you provide a JSON document to help me better understand your paper? Thank you very much.

BTW, do you plan to submit this work to a conference or journal? It looks like a truly outstanding piece of research.

**Comment by Fyc-Ethan** [comment]
Dear xypkent,

First of all, thank you for your reply, looking forward to your training code.

Actually, I've run into another problem. Please help me out~

New problem: 

When I run your Quick Start code from HuggingFace, I found that you used the `Qwen2VLForConditionalGeneration` to load the Qwen2.5_VL.

But an error occurred when I loaded the runtime: You are using a model of type qwen2_5_vl to instantiate a model of type qwen2_vl. This is not supported for all configurations of models and can yield errors.

Therefore, could you please confirm whether it's possible to use `Qwen2_5_VLForConditionalGeneration` to load Qwen2.5_VL? Or do I need to load Qwen2_VL?


Best regards
